test/handler.go

```diff
@@ -84,22 +88,23 @@ type Server struct {
 
 // NewServer returns a test server running on a random port.
 func NewServer() *Server {
-	handler, err := NewHandler()
-	if err != nil {
-		panic(err)
-	}
-	s := &Server{
-		Handler: handler,
-	}
-	s.Server = httptest.NewServer(s.Handler.Handler)
-
-	// Handler test messages can no-op.
-	s.Handler.API.Broadcaster = pilosa.NopBroadcaster
-	// Create a default cluster on the handler
-	s.Handler.API.Cluster = NewCluster(1)
-	s.Handler.API.Cluster.Nodes[0].URI = s.HostURI()
-
-	return s
+	return &Server{}
+	//handler, err := pilosa.NewHandler()
```

Comment:
test.NewServer will probably be deleted, but we'll keep the commented implementation here in case we need to reason about it while we fix the last few tests that use it.